LLMonFHIR/Settings/Prompt.swift

```diff
@@ -17,10 +17,8 @@ enum Prompt: String {
     
     case interpretMultipleResources = "prompt.interpretMultipleResources"
     
-    
```

Comment:
We have a style guide in place that we have two empty lines between blocks of elements in a type, such as case statements and static properties. It would be great if you can re-add the lines here.